### PR TITLE
Added revised version of the no-new-privs post.

### PR DESCRIPTION
**Pull request by jberkus**
This is the revised version of @mrunalp 's post about no-new-privileges.  It is not ready to go yet.  There's a number of questions:

* @mrunalp, I had to revise the post significantly, please verify that I didn't add any errors
* @mrunalp it's your patch to Docker, no?  Shouldn't we credit you by name?  Who else worked on it?
* @rhatdan, @jlebon, @jasonbrooks please check sample code, thanks!

### CODE REVIEW

File: source/blog/2016-03-17-no-new-privs-docker.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Docker credentials store
```
Comment (mrunalp) [inline source/blog/2016-03-17-no-new-privs-docker.md]: Metadata doesn't look right.

**Comment by mrunalp** [comment]
@jberkus Yes, I worked on the patches to OCI spec, runc and docker. @rhatdan helped with the selinux policy changes required for it.

**Comment by rhatdan** [comment]
Change the Red Hat staff ... Intro to I.

Then add a paragraph:

Turning on no_new_privs actually stopped the SELinux transition from the docker daemon type `docker_t` to the container type, `svirt_lxc_net_t`.  no_new_privs only allows  SELinux transitions from one type to another iff the target type as a complete subset of the source type.  Dan Walsh worked on the
SELinux policy for docker to fix this. With the latest policy in Fedora 24, no_new_privs and SELinux work well together.  We will be back porting these fixes to RHEL when we ship docker support for no_new_privs.

**Comment by jberkus** [comment]
OK, updated with changes.  @bproffitt, this should be ready to go unless Dan or Mrunal object.

**Comment by jberkus** [comment]
Crap, closed this by accident.  @bproffitt, if you can publish this tommorrow?

**Pull request merged**